### PR TITLE
Bump to version 0.0.2

### PR DESCRIPTION
**Pull request by Huy-Ngo**

### CODE REVIEW

**Review by 9a24f0 — DISMISSED** [review]

**Comment by McSinyx** [comment]
Regarding project's metadata, just realized that we should be using `Operating System :: POSIX` instead of `Operating System :: OS Independent` because `crypt` doesn't work on e.g. Windows.

**Review by McSinyx — APPROVED** [review]
:tada:

**Comment by Huy-Ngo** [comment]
@9a24f0 Please re-approve since I updated :p

**Comment by Huy-Ngo** [comment]
~~Fine. I'll do it myself~~

File: pyproject.toml [hunk 1]
```diff
@@ -18,8 +18,7 @@ classifiers = [
     'Intended Audience :: End Users/Desktop',
     'License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)',
     'Natural Language :: English',
-    'Natural Language :: Vietnamese',
```
Comment (9a24f0) [inline pyproject.toml:21]: why you remove this :(

Comment (McSinyx) [reply pyproject.toml:21]: cuz this release doesn't support vietnamse

**Review by 9a24f0 — APPROVED** [review]
LGTM!

**Pull request merged**